base/src/Utils.cpp

```diff
@@ -148,6 +149,12 @@ cv::Mat Utils::getMatHeader(RawImageMetadata* metadata)
 	return cv::Mat(metadata->getHeight(), metadata->getWidth(), metadata->getType(), static_cast<void*>(&data), metadata->getStep());
 }
 
+cv::Mat Utils::getMatHeader(RawImagePlanarMetadata* metadata)
+{
+	uint8_t data = 0;
+	return cv::Mat(metadata->getHeight(0) + metadata->getHeight(0) / 2, metadata->getWidth(0), CV_8UC1, static_cast<void*>(&data));
```

Comment:
This is making an assumption of YUV420... not a good idea.

Reply:
It was a mistake, i've resolved it.